.gitignore

```diff
@@ -2,3 +2,5 @@
 node_modules
 phpunit.xml
 vendor
+.idea
```

Comment:
Please don't add these files to the Cachet `.gitignore`. These should be done on your preferences only.